specification/web/resource-manager/Microsoft.Web/stable/2024-11-01/WebApps.json

```diff
@@ -29212,6 +29212,10 @@
               "description": "<code>true</code> to enable client affinity; <code>false</code> to stop sending session affinity cookies, which route client requests in the same session to the same instance. Default is <code>true</code>.",
               "type": "boolean"
             },
+            "clientAffinityProxyEnabled": {
```

Comment:
>clientAffinityProxyEnabled [](http://example.com/codeflow?start=13&length=26)

Could you update relevant examples too ?

Reply:
Examples added